12272020/root/opt/joy2key/example.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Example whiptail dialog for ArkOS using RetroPie joy2key.py loader
+# by ridgek
+###########
+# PREFLIGHT
+###########
+RETROPIE_HELPERS="/opt/joy2key/RetroPie-Setup/scriptmodules/helpers.sh"
+
+if [ -f "${RETROPIE_HELPERS}" ]; then
+	source "${RETROPIE_HELPERS}"
+
+	# Required by joy2keyStart
+	scriptdir="/opt/joy2key/RetroPie-Setup"
+	# Arguments to pass to joy2keyStart (left, right, up, down, return, space, esc)
+	keyconfig=(kcub1 kcuf1 kcuu1 kcud1 0x0a 0x20 0x1b)
```

Comment:
This would need to change to the following to allow enter and escape using a and b.

`keyconfig=(kcub1 kcuf1 kcuu1 kcud1 0x0d 0x1b)`

Reply:
Isn't new line 0x0a the proper Linux convention instead of carriage return 0x0d? (btw, are you working on Windows? I've noticed when working on Update-RG351P.sh that git always complains about "no new line character" on the last line and I can never stop Vim from changing the very last line, lol)

Reply:
I usually work in Windows using notepad++ in unix mode.  If I don’t do that, the script doesn’t run at all because each line in the script would end with a ^M.  I always keep my Linux VM running as well as I hoo back and forth between it and Windows.

Reply:
As far as 0a being proper carriage return, it’s supposed to be but it didn’t move through the whiptail on my end without selecting OK.  0d does.

Reply:
Carriage return it is, then! I'll commit all the above changes in the next push